### ISSUE TITLE
Support for mimic joints

### ISSUE THREAD

**Issue by costashatz** [leggedrobotics/raisimLib#37]
Thanks for the nice simulator.

Are there any plans for supporting mimic joints (that URDF supports)? We really need this feature. Thanks a lot! If needed I can provide test URDF files..

**Comment by jhwangbo** [comment]
Hi, costashatz. Probably around May. I probably need a full week for this but I dont have that much time at the moment.

**Comment by costashatz** [comment]
@jhwangbo thanks for the fast reply.

I will ping you again in May then..

**Comment by costashatz** [comment]
@jhwangbo a gentle reminder for making this feature available! Thanks again!

**Comment by jhwangbo** [comment]
Yes, this is my next feature to implement. Not sure how long it will take

**Comment by costashatz** [comment]
> Yes, this is my next feature to implement.

Great!

> Not sure how long it will take

Ping this issue when you are done (so that I get notified). Thanks!
